deploy/crds/managed_v1alpha2_subjectpermission_cr.yaml

```diff
@@ -1,13 +1,10 @@
 apiVersion: managed.openshift.io/v1alpha1
 kind: SubjectPermission
 metadata:
-  name: example-subjectpermission
+  name: example2-subjectpermission
```

Comment:
You'll also need to change the name of the CR file

Reply:
yes agreed James made a comment about that as well